src/praisonai-code/praisonai_code/cli/commands/usage.py

```
@@ -0,0 +1,173 @@
+    otherwise the current project store is merged with the global default
+    store (mirrors ``session list`` semantics).
+    """
+    from ..state.project_sessions import (
+        get_project_session_store,
+        list_project_sessions,
+    )
+
+    if project:
+        try:
+            sessions = get_project_session_store(project_id=project).list_sessions(limit=1000)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)